src/test/java/APIBoundaryTest/SpringSetupTest.java

```diff
@@ -0,0 +1,27 @@
+package APIBoundaryTest;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+import java.io.IOException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.URI;
+import java.net.http.HttpResponse;
+
+
+public class SpringSetupTest {
+    @Test
+    public void testApiSetup() throws IOException, InterruptedException {
```

Comment:
at first glance i thought this meant senso api, can you add documentation or rename the method to make that clear you are testing our back-end's api is running correctly on EC2?